pkg/controller/services.go

```diff
@@ -278,7 +278,7 @@ func (cont *AciController) fabricPathForNode(name string) (string, bool) {
 	sz := len(cont.nodeOpflexDevice[name])
 	for _, device := range cont.nodeOpflexDevice[name] {
 		if device.GetAttrStr("state") == "connected" {
-			cont.fabricPathLogger(device.GetAttrStr("hostName"), device).Info("Processing fabric path for node ",
+			cont.fabricPathLogger(device.GetAttrStr("hostName"), device).Debug("Processing fabric path for node ",
```

Comment:
In this patch lets just restore this to Info and we can consider a change in a separate patch.